gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobLauncher.java

```diff
@@ -205,6 +210,13 @@ protected void runWorkUnits(List<WorkUnit> workUnits) throws Exception {
       this.eventSubmitter.submit(countEventBuilder);
       LOGGER.info("Emitting WorkUnitsCreated Count: " + countEventBuilder.getCount());
 
+      MetricContext
+          metricContext = Instrumented.getMetricContext(ConfigUtils.configToState(ConfigFactory.empty()), getClass());
+      String workUnitCountName = MetricRegistry.name(this.jobContext.getJobState().getProp(ConfigurationKeys.FLOW_GROUP_KEY),
+          this.jobContext.getJobState().getProp(ConfigurationKeys.FLOW_NAME_KEY), "workunits");
```

Comment:
Let's make the name a bit more descriptive than "workunits", maybe just reuse the `JobEvent.WORK_UNITS_CREATED` name that's used for the events.